account/user.go

```diff
@@ -94,22 +93,10 @@ func (m *GormUserRepository) Load(ctx context.Context, id uuid.UUID) (*User, err
 	return &native, errs.WithStack(err)
 }
 
-// Exists returns true|false where an object exists with an identifier
-func (m *GormUserRepository) Exists(ctx context.Context, id uuid.UUID) (bool, error) {
+// Exists returns true|false whether an user exists with a specific identifier
```

Comment:
I think this reads better:  `Exists returns true if a user exists with the given ID`